t/16-utils-runcmd.t

```diff
@@ -3,13 +3,12 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 use Test::Most;
+use Mojo::Base -signatures;
 
 use FindBin;
 use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
 use Data::Dumper;
-use OpenQA::Git;
 use OpenQA::Utils;
-use OpenQA::Task::Needle::Save;
```

Comment:
Those modules are still used.

Reply:
All tests pass for me without the imports, is there a benefit in declaring them anyway ?

Reply:
Yes, keep them. Every module used in a file should be imported explicitly. Tests still pass because the modules are still indirectly imported by other meanss.